sstables/types.hh

```diff
@@ -509,8 +509,29 @@ struct run_identifier {
     auto describe_type(sstable_version_types v, Describer f) { return f(id); }
 };
 
+// Types of large data statistics.
+//
+// Note: For extensibility, never reuse an identifier,
+// only add new ones, since these are stored on stable storage.
+enum class large_data_type : uint32_t {
+    partition_size = 0,     // partition size, in bytes
+    row_size = 1,           // row size, in bytes
+    cell_size = 2,          // cell size, in bytes
+    rows_in_partition = 3,  // number of rows in a partition
+};
```

Comment:
This should be added to the docs.

Recommend starting from 1, so that there is always a non-zero value that stands out.

Reply:
fixed in v5